src/Sulu/Bundle/MediaBundle/Domain/Event/MediaCropModifiedEvent.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\MediaBundle\Admin\MediaAdmin;
+use Sulu\Bundle\MediaBundle\Entity\Collection;
+use Sulu\Bundle\MediaBundle\Entity\FileVersionMeta;
+use Sulu\Bundle\MediaBundle\Entity\MediaInterface;
+
+class MediaCropModifiedEvent extends DomainEvent
```

Comment:
think we should call this `MediaCropChangedEvent` if we use `MediaPreviewImageChangedEvent` for the preview image 🙂

Reply:
okay not sure about this anymore, i guess it will be confusing when to use `Change` and when to use `Modify` 🙈 
maybe wo should just always use `Modify`?

Reply:
I don't agree on that one ... I would use `changed` only, if you switch from one item in a predefined list to another ... For the preview image, the predefined list is the media gallery. So you just choose one image and change your selection later. But for crops, you really modify the crop ... The user is moving the square around and setting the exact position and size of that square as crop

Reply:
Ah, didn't read the last comment ... my comment was related to the first one 🙈

Reply:
> I would use changed only, if you switch from one item in a predefined list to another ... For the preview image, the predefined list is the media gallery

Yeah I see that the differentiation would make sense in this case.. But I am afraid that the terms are too similar and once both terms are used in the codebase, somebody will use the wrong term unintentionally and we will end up with an inconsistent naming 🙈 
To prevent this, we could use a more explicit term like `preview_image_assigned`. But I am not sure if this is worth the effort. I think the simplest thing would be just using `modified` everywhere